sections/featured-product.liquid

```diff
@@ -750,4 +750,4 @@
     }
   ]
 }
-{% endschema %}
+{% endschema %}
```

Comment:
To avoid these types of diffs in PRs, add an empty line at the end of each file.  You can enable this as a setting in VS Code to not need to think about it

<img width="786" alt="Screen Shot 2022-02-04 at 2 37 47 PM" src="https://user-images.githubusercontent.com/60230011/152592364-cd4b65e0-ed13-40f2-ac6b-f0e56d643ab2.png">

```suggestion
{% endschema %}

```